our-work/workshops-and-talks/how-to-pr.yml

```diff
@@ -0,0 +1,19 @@
+title: "Collaborative Coding: How To PR with GitHub"
+description:
+  In this hands-on workshop we covered collaborative coding, making and understanding branches, and making code suggestions with pull requests.
+slug: how-to-pr
+languages:
+  - git
+  - markdown
+tags:
+  - collaboration
+people:
+  - name: John Gerrard Holland
+    github_user: hollandjg
+groups:
+  - dscov
+links:
+  - category: website
+    url: https://github.com/dscov-tutorials/how-to-pr
+  - category: video
+    url: https://drive.google.com/file/d/1LK_mf6r6jo5ojcIjcg3kAMr4hu534rVi/view?usp=drive_web
```

Comment:
Hmm, we need access to this?

Perhaps put the video on [Panopto](https://brown.hosted.panopto.com/Panopto/Pages/Auth/Login.aspx?ReturnUrl=https%3A%2F%2Fbrown.hosted.panopto.com%2FPanopto%2FPages%2FSessions%2FList.aspx) instead?

Reply:
Thanks for the feedback, Ellen. I've not managed to get the release form for the video, so it can't be posted right now. I've updated the PR to remove the inaccessible video link.